node/src/accountant/big_int_db_processor.rs

```diff
@@ -0,0 +1,1613 @@
+// Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+
+use crate::accountant::big_int_db_processor::ByteOrder::{High, Low};
+use crate::accountant::big_int_db_processor::UserDefinedFunctionError::InvalidInputValue;
+use crate::accountant::big_int_db_processor::WeiChange::{Addition, Subtraction};
+use crate::accountant::checked_conversion;
+use crate::accountant::payable_dao::PayableDaoError;
+use crate::accountant::receivable_dao::ReceivableDaoError;
+use crate::database::connection_wrapper::ConnectionWrapper;
+use crate::sub_lib::accountant::WEIS_OF_GWEI;
+use crate::sub_lib::wallet::Wallet;
+use itertools::Either;
+use masq_lib::utils::ExpectValue;
+use rusqlite::functions::{Context, FunctionFlags};
+use rusqlite::Error::UserFunctionError;
+use rusqlite::{Connection, Error, Statement, ToSql, Transaction};
+use std::fmt::{Debug, Display, Formatter};
+use std::iter::once;
+use std::marker::PhantomData;
+use std::ops::Neg;
+
+#[derive(Debug)]
+pub struct BigIntDbProcessor<T: DAOTableIdentifier> {
+    phantom: PhantomData<T>,
+}
+
+impl<T: DAOTableIdentifier> BigIntDbProcessor<T> {
+    pub fn execute<'a>(
+        &self,
+        conn: Either<&dyn ConnectionWrapper, &Transaction>,
```

Comment:
You can simplify here: Transaction is no longer a thing. (I think.)

Reply:
We will talk about this later again. Transaction still has its place in the code, just elsewhere than you think.